.travis.yml

```diff
@@ -0,0 +1,18 @@
+branches:
+  only:
+    - master
+cache:
+  directories:
+    - vendor
+sudo: required
+services:
+  - docker
+install:
+  - make bootstrap
+script:
+  - make test
+deploy:
+  provider: script
+  script: make ci-deploy
```

Comment:
Should this be an external script like _scripts/deploy.sh in the other components? The Makefile target could just call that...